drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/PhantomJSDriverFactory.java

```diff
@@ -35,17 +33,14 @@
  * PhantomJSDriver.
  *
  * @author <a href="kpiwko@redhat.com">Karel Piwko</a>
- *
+ * @author <a href="mjobanek@redhat.com">Matous Jobanek</a>
```

Comment:
I was thinking about removing author tags from every class :) WDYT? (nothing for this PR - just to open the discussion)